pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>commons-io</artifactId>
             <version>2.4</version>
         </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>1.1.1</version>
```

Comment:
Actually I didn't add any logger call, but I got a kind of jar hell on gocd-server 15.2. I just tried to reproduce, but I couldn't, so I'm removing this.